### ISSUE TITLE
High contrast issue with scroll-able panel control

### ISSUE THREAD

**Issue by v-abkhot** [microsoft/fluentui#13205]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/v-abkhot/pen/yLYjvaz

steps are as follows : - open code pen go to scrollable panel -> hover the list items those will get highlighted -> now turn on highcontrast mode -> go to scrollable panel hover on item list you will found that items are not getting highlighted on mouse hover on items.

these are the steps to repro.

#### Actual behavior:

In scroll-able-panel we can focus on every item which is visible in normal mode.
but it is not visible in high-contrast mode

#### Expected behavior:

In scroll-able-panel we can focus on every item which is visible in normal mode.
but it should visible in high-contrast mode

**Comment by aneeshack4** [comment]
@v-abkhot thanks for reaching out to us. I do see the repro:

**Comment by v-abkhot** [comment]
after selection it is visible as in your screen shot.

but if i just hover on the items it is not visible 
as in my screen shot

![image](https://user-images.githubusercontent.com/63849538/82232903-d9de2500-994c-11ea-8eff-b69f8da7533f.png)

**Comment by ecraig12345** [comment]
Closing since this is a duplicate of #13066. Please don't open duplicate issues (and sorry we didn't follow up sooner on the original).
